ui/components/ui/button/button.stories.js

```diff
@@ -48,13 +48,6 @@ export default {
     className: { control: 'text' },
     onClick: { action: 'clicked' },
   },
-  args: {
-    disabled: false,
-    large: false,
-    submit: false,
-    className: '',
-    rounded: true,
-  },
```

Comment:
Uses defaults set in component instead of overriding the story

<img width="1512" alt="Screenshot 2025-05-14 at 8 25 32 AM" src="https://github.com/user-attachments/assets/dd4caf71-1fc6-4445-b5fc-a1634b7d9fac" />